src/components/pages/Home.tsx

```diff
@@ -109,7 +109,11 @@ export default function HomePage(): ReactElement {
         action={
           <Button
             style="text"
-            to="/search?priceType=pool&sort=liquidity&sortOrder=desc"
+            to={
+              screen.width > 480
```

Comment:
this checks for the device's screen width regardless of browser viewport size, but we should always check for the actual available viewport. Which rules out using the Screen API. If you absolutely have to check for viewport size in JS, use [`window.matchMedia`](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia). In general we should try to avoid viewport size detection in JS cause CSS is just better for that, e.g. gives viewport resize handlers for free.

But adapting the search string based on viewport might be not best solution. If the intention was to hide searchbar on smaller viewports then this should be done over in search page component